Test/DependencyCollector/FunctionalTests/FuncTest/Helpers/DeploymentAndValidationTools.cs

```diff
@@ -197,11 +197,17 @@ public static void CleanUp()
         public static void Validate(
             TelemetryItem<RemoteDependencyData> itemToValidate,
             TimeSpan accessTimeMax,
-            bool successFlagExpected)
+            bool successFlagExpected,
+            string resultCodeExpected = "DontCheck")
```

Comment:
string.Empty will be the value for clientside exception, so can't just pass that as default.